project/src/components/comment-submission-form/comment-submission-form.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+
+function CommentSubmissionForm(): JSX.Element {
+
+  const [formData, setFormData] = useState({
+    rating: '',
+    review: '',
+  });
+
+  const fieldChangeHandle = (evt: { target: { name: string; value: string } }) => {
+    const {name, value} = evt.target;
+    setFormData({...formData, [name]: value});
+  };
```

Comment:
не стоит так делать, без крайней нужды, все браузерные события уже типизированы , если не знаешь какой тип использовать гугли )
```suggestion
  const fieldChangeHandle = (evt: React.ChangeEvent<HTMLInputElement>) => {
    const {name, value} = evt.target;
    setFormData({...formData, [name]: value});
  };
```

Reply:
с остальными обработчиками аналогично